test/fields/e2e.spec.ts

```diff
@@ -1360,18 +1360,16 @@ describe('fields', () => {
     test('should display formatted date in useAsTitle', async () => {
       await page.goto(url.list)
       await page.locator('.row-1 .cell-default a').click()
-      await expect(page.locator('.collection-edit .doc-header__title.render-title')).toContainText(
```

Comment:
If this `collection-edit` class was missing, it should probably be placed back onto the div instead of removed from the test.